internal/middleware/logger.go

```diff
@@ -31,6 +31,8 @@ func LoggerMiddleware(rootLogger *zerolog.Logger) func(next http.Handler) http.H
 				Logger()
 			t1 := time.Now()
 
+			logger.Debug().Msg("Started request")
```

Comment:
I think method and URL is in the annotation vars already, so nothing on top of that is needed I guess.

Reply:
Yes it is and the general idea of structured logging is to only put message without copies of these fields, yet, I would suggest to format it the same way as in "Completed..." message, I find it very useful when searching logs by eyes without all the fields enabled in Kibana.

Reply:
I find it very useful, this is subjective I have to admit:

![image](https://user-images.githubusercontent.com/49752/212924983-2e8bddd7-cbfa-4fef-937f-e5b57735e49e.png)